tests/tests_sign_workflow/test_signer.py

```diff
@@ -54,6 +55,7 @@ def test_accepted_file_types_sig(self, git_repo):
             call(os.path.join("path", "the-module.module"), ".sig"),
             call(os.path.join("path", "the-tar.tar.gz"), ".sig"),
             call(os.path.join("path", "something-1.0.0.0.jar"), ".sig"),
+            call(os.path.join("path", "opensearch_sql_cli-1.0.0-py3-none-any.whl"), ".asc"),
```

Comment:
Why is this one using asc not sig?

Thanks.

Reply:
Yup, fixed